src/Statement.py

```diff
@@ -46,8 +56,11 @@ def is_fee(self, booking_type):
 
     def get_date(self):
         """ get the date of the statement """
+        date_to_parse = self._statement[self._config.get_booking_date()]
+        if date_to_parse == "":
+            return datetime(1970, 1, 1).date()
         return datetime.strptime(
-            self._statement[self._config.get_booking_date()],
+            date_to_parse,
             self._config.get_booking_date_format(),
         ).date()
```

Comment:
Hm. would this be cleaner?
```
        if self._statement[self._config.get_booking_date()]:
            statement_date = datetime.strptime(
                self._statement[self._config.get_booking_date()], self._config.get_booking_date_format()
            ).date()
        else:
            statement_date = datetime(1970, 1, 1).date()
        return statement_date
```

Could you also add a sample entry to the mintos test data to verify that an empty entry is correctly processed? :)

Reply:
Yes that looks a lot better, thanks!